src/transformers/convert_slow_tokenizer.py

```diff
@@ -583,6 +583,7 @@ def normalizer(self, proto):
         list_normalizers = []
         if self.original_tokenizer.do_lower_case:
             list_normalizers.append(normalizers.Lowercase())
+            list_normalizers.append(normalizers.Strip())
```

Comment:
This line needs to be added - otherwise behavior differs from slow tokenizer: 
```
AssertionError: Lists differ: ['▁hello', '!', 'how', '▁are', '▁you', '?', '▁'] != ['▁hello', '!', 'how', '▁are', '▁you', '?']
```

Reply:
Shouldn't `Strip` also be applied even when `do_lower_case` is False? When I compare the slow tokenizer to the fast one, the slow one removes the whitespace and the fast one doesn't.

Reply:
@mingboiz did you see my comment?

Reply:
yes, have been swamped with work and didn’t have time to check this weekend

Reply:
> yes, have been swamped with work and didn’t have time to check this weekend

Understandable! Take your time

Reply:
@nbroad1881 you are quite right, thank you for catching it, I will be adding another test for this